packages/app-desktop/gui/WindowCommandsAndDialogs/commands/linkToNote.ts

```diff
@@ -0,0 +1,37 @@
+import CommandService, { CommandRuntime, CommandDeclaration, CommandContext } from '@joplin/lib/services/CommandService';
+import { _ } from '@joplin/lib/locale';
+import { Mode } from '../../../plugins/GotoAnything';
+import { GotoAnythingOptions, UiType } from './gotoAnything';
+import { ModelType } from '@joplin/lib/BaseModel';
+import Logger from '@joplin/utils/Logger';
+import markdownUtils from '@joplin/lib/markdownUtils';
+
+const logger = Logger.create('linkToNote');
+
+export const declaration: CommandDeclaration = {
+	name: 'linkToNote',
+	label: () => _('Link to note...'),
+};
+
+export const runtime = (): CommandRuntime => {
+	return {
+		execute: async (_context: CommandContext) => {
+			const options: GotoAnythingOptions = {
+				mode: Mode.TitleOnly,
+			};
+			const result = await CommandService.instance().execute('gotoAnything', UiType.ControlledApi, options);
+			if (!result) return result;
+
+			if (result.type !== ModelType.Note) {
+				logger.warn('Retrieved item is not a note:', result);
+				return null;
+			}
```

Comment:
Since this is only for notes, it may make sense to change the help text in the Goto Anything dialog in this case (e.g. "search for a note to link").

Reply:
Indeed I forgot about this help tooltip. Thanks, I will update this